README.md

```diff
@@ -1435,7 +1408,7 @@ propose the following schema that satisfies the consolidated requirements of:
     "share": "1, 2, 3, ..., 100",
     "validFrom": { "@type": "http://schema.org/Date" },
     "validTo": { "@type": "http://schema.org/Date" },
```

Comment:
Note that these instances of `validTo` and `validFrom` should be fixed with #14.